### ISSUE TITLE
Not working after flutter 3.24.3 update

### ISSUE THREAD

**Issue by luccaslyra** [DavBfr/dart_pdf#1766]
it's a flutter web app testing on chrome. Before flutter 3.24.3 it was working fina, but now i got this erro:

errors.dart:296 Uncaught (in promise) DartError: TypeError: null: type 'Null' is not a subtype of type 'JSObject'
at Object.throw_ [as throw] (errors.dart:296:3)
at Object._failedAsCheck (rti.dart:1405:3)
at dart_rti.Rti.new._generalAsCheckImplementation (rti.dart:1383:3)
at Object.getProperty (js_util_patch.dart:81:5)
at printing_web.PrintingPlugin.new.layoutPdf (printing_web.dart:196:32)
at layoutPdf.next ()
at async_patch.dart:45:50

**Comment by samirpegado** [comment]
Same issue here.

DartError: TypeError: null: type 'Null' is not a subtype of type 'JSObject'
dart-sdk/lib/_internal/js_dev_runtime/private/ddc_runtime/errors.dart 296:3  throw_
errors.dart:296
dart-sdk/lib/_internal/js_shared/lib/rti.dart 1405:3                         _failedAsCheck
rti.dart:1405
dart-sdk/lib/_internal/js_shared/lib/rti.dart 1383:3                         _generalAsCheckImplementation
rti.dart:1383
dart-sdk/lib/_internal/js_shared/lib/js_util_patch.dart 81:5                 getProperty
js_util_patch.dart:81
packages/printing/printing_web.dart 196:32                                   layoutPdf
printing_web.dart:196
dart-sdk/lib/_internal/js_dev_runtime/patch/async_patch.dart 45:50           <fn>
async_patch.dart:45
dart-sdk/lib/async/zone.dart 1661:54                                         runUnary
zone.dart:1661
dart-sdk/lib/async/future_impl.dart 163:18                                   handleValue
future_impl.dart:163
dart-sdk/lib/async/future_impl.dart 861:44                                   handleValueCallback
future_impl.dart:861
dart-sdk/lib/async/future_impl.dart 890:13                                   _propagateToListeners
future_impl.dart:890
dart-sdk/lib/async/future_impl.dart 666:5                                    [_completeWithValue]
future_impl.dart:666
dart-sdk/lib/async/future_impl.dart 736:7                                    callback
future_impl.dart:736
dart-sdk/lib/async/schedule_microtask.dart 40:11                             _microtaskLoop
schedule_microtask.dart:40
dart-sdk/lib/async/schedule_microtask.dart 49:5                              _startMicrotaskLoop
schedule_microtask.dart:49
dart-sdk/lib/_internal/js_dev_runtime/patch/async_patch.dart 181:7           <fn>

**Comment by muazam99** [comment]
same issue here

**Comment by Lailan** [comment]
Same issue here.

**Comment by emanuelmarlon** [comment]
I also have the same error in flutter version 3.24.2, can someone help?

**Comment by DavBfr** [comment]
Which version of the `printing` and `web` packages do you use?

**Comment by luccaslyra** [comment]
Hi,

web: ^1.1.0
printing: ^5.13.3

**Comment by DavBfr** [comment]
It's fixed in the master branch. Let me publish version 5.13.4 with that.

**Comment by tmpsantos** [comment]
Thank you @DavBfr for quickly acting on this! 🏆

**Comment by DavBfr** [comment]
Don't hesitate to [Buy Me A Coffee](https://www.buymeacoffee.com/JORBmbw9h).

**Comment by emanuelmarlon** [comment]
Have you managed to solve it yet? Help me

**Comment by luccaslyra** [comment]
Thanks @DavBfr, you're awesome !

**Comment by luccaslyra** [comment]
@emanuelmarlon yes, it was the printing dependencie, i just updated to latest one(the one they just released 5.13.4)
